src/main/java/by/itechart/api/dto/UpdateUserDTO.java

```diff
@@ -0,0 +1,41 @@
+package by.itechart.api.dto;
+
+import io.swagger.annotations.ApiModelProperty;
+import lombok.Data;
+
+@Data
+public class UpdateUserDTO {
+    @ApiModelProperty(
+            value = "User's first name",
```

Comment:
Since it's user's update info I think it would be more descriptive to say `User's new first name` and all update fields are optional so it would be better to set `required = true`

Reply:
If all fields are optional then why we need to set `required = true`? ;) Maybe you supposed to remove the required statement?

Reply:
I mean `required = false` or remove it, it's `false` by default